source/core/wifi_ctrl_queue_handlers.c

```diff
@@ -3011,6 +3011,32 @@ static void process_monitor_init_command(void)
     free(data);
 }
 
+void process_send_action_frame_command(void *data, unsigned int len)
+{
+    action_frame_params_t *params;
+
+    if (data == NULL) {
+        wifi_util_error_print(WIFI_CTRL, "%s:%d NUll data Pointer\n", __func__, __LINE__);
+        return;
+    }
+
+    if (len < sizeof(action_frame_params_t) + 1) {
+        wifi_util_error_print(WIFI_CTRL, "%s:%d Invalid parameter size \r\n", __func__, __LINE__);
+        return;
+    }
+
+    params = (action_frame_params_t *)data;
+
+    if (wifi_sendActionFrame(params->ap_index, params->dest_addr, params->frequency,
```

Comment:
Based on the discussion on this PR, it may be better to use `wifi_hal_send_mgmt_frame` instead of `wifi_sendActionFrame` which could be unimplemented.

https://github.com/rdkcentral/rdk-wifi-hal/pull/13

Reply:
@bcarlson-dev, it is better to use wifi_sendActionFrame as that is the interface which is declared in wifi_hal_ap.h instead of directly using wifi_hal_send_mgmt_frame which is in wifi_hal_priv.h

Reply:
Makes sense, just wanted to confirm